### ISSUE TITLE
Feature request: simultaneous key presses

### ISSUE THREAD

**Issue by Keek** [xmichelo/Beeftext#152]
I would like to request simultaneous key presses in combos, to trigger functionality in programs where the combo is expanded, e.g.: {Ctrl + Enter}

**Comment by xmichelo** [comment]
Hi @Keek.

Thanks you for your suggestion. I understand the need, and I will keep this feature request open, but with low priority. This means it's not going to be implemented soon.

If you really need this feature, I invite you to consider looking at commercial alternatives (such as [PhraseExpress](https://www.phraseexpress.com/).

**Comment by majorthorn** [comment]
@xmichelo Just to make sure that I ask before creating a new feature request would this be where typing a keyword and nothing happens unless pressing {SHIFT + ENTER}? Or am I reading this wrong?

**Comment by xmichelo** [comment]
@majorthorn no, this one is a request to be able to include keys with modifiers - or special keys like tab - in a combo. 

What you are requesting is actually already implemented. If you go in the Preferences dialog, have a look at the section named 'Combo triggering'. If you set it to 'manual', combo replacement does not take place until you type a keyboard shortcut. The default is `Ctrl+Alt+Shift+B`, but you are free to change it.

**Comment by majorthorn** [comment]
@xmichelo Great! Thank you. I couldn't find that setting the other day. If it was a snake it would have bit me.

**Comment by wilhelmmedetz** [comment]
Hi, I want to support this request.
The most typical use case for me would be to insert `#{clipboard}` by pressing `Ctrl+Shift+Insert`
